build/yamls/flow-aggregator.yml

```diff
@@ -203,15 +203,40 @@ data:
 
       # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
       #tlsMinVersion:
+
+    # clickHouse contains ClickHouse related configuration options.
+    #clickHouse:
+      # DB name containing Antrea table "flows".
+      #database: "default"
+      # URL to ClickHouse DB.
+      #dbUrl: "tcp://clickhouse-clickhouse.flow-visibility.svc:9000"
```

Comment:
Hi Shawn, when I was doing some local tests, I find this `#dbUrl` needs to be uncommented or I will get such error:
` main.go:56] Failed to validate args: IPFIX flow collector address or ClickHouse should be configured`
Could you please fix it? Thanks!

Reply:
I assume it's because the entire `clickHouse` option dict is empty. we need to have some config declared to turn ch on, otherwise there's no measure to turn it off (even if we added most default values). the easiest way atm is by indicating debug: false to make the sub conf non-empty dict. do you think we should add an explicit "enabled" param for this?

Reply:
I have uncommented `#clickHouse:` on line 208. I assume it could enable the `clickhouse` option like what we did for flow exporter.

Reply:
added an explicit switch in the latest commit `clickHouse.enabled`. should be more intuitive for users who want to turn on this feature.